### PR TITLE
Vshnmanaged redis docs

### PR DESCRIPTION
**Pull request by TheBigLee**

### CODE REVIEW

File: docs/modules/ROOT/pages/vshn-managed/redis/index.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+= Managed by VSHN - Redis
```
Comment (tobru) [inline docs/modules/ROOT/pages/vshn-managed/redis/index.adoc:1]: Can we call it "Redis by VSHN"? (also for PostgreSQL then)

**Review by tobru — COMMENTED** [review]
There are two commits in this PR which are completely unrelated. Is that by intention?

**Comment by TheBigLee** [comment]
> There are two commits in this PR which are completely unrelated. Is that by intention?

The issue is, that the API is generated from the component. Since nobody did that for the backup feature, I have split it in two commits. However, it makes more sense to split it in two PRs.

File: docs/modules/ROOT/pages/vshn-managed/redis/index.adoc (hunk 1 above)
Comment (TheBigLee) [reply docs/modules/ROOT/pages/vshn-managed/redis/index.adoc]: Fixed. For Postgresql see: #30

**Review by tobru — APPROVED** [review]

**Pull request merged**